zendesk/search.go

```diff
@@ -68,7 +68,8 @@ func OrganizationFilter(organizationID int) Filters {
 // SearchTickets leverages the unified search api to return tickets
 //
 // Zendesk Core API docs: https://developer.zendesk.com/rest_api/docs/support/search
-func (c *client) SearchTickets(term string, options *ListOptions, filters ...Filters) (*TicketSearchResults, error) {
+// Zendesk Side load API docs: https://developer.zendesk.com/api-reference/ticketing/ticket-management/search/#side-loading
+func (c *client) SearchTickets(term string, options *ListOptions, sideload SideLoad, filters ...Filters) (*TicketSearchResults, error) {
```

Comment:
should "SideLoad" be an array? it seems like you could have multiple options.

Reply:
yes I do, the first time I did it was giving me errors but now is working well with an array.. I think was my mistake when testing this code. Added the support for arrays in sideload